### PR TITLE
add importTNChecker

### PR DESCRIPTION
**Pull request by dtolb**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -22,5 +24,13 @@ module.exports = {
   DiscNumber: require("./discNumber"),
   LsrOrder: require("./lsrOrder"),
   Lidbs: require("./lidbs"),
-  Dlda: require("./dlda")
+  Dlda: require("./dlda"),
+  ImportTnChecker: require("./importTnChecker")
 };
+
+for (const property in module.exports) {
```
Comment (Aubron) [inline lib/index.js]: Clever girl

Comment (Aubron) [reply lib/index.js]: 79e1c5b7b6cfd4ad9000b2213ab1daef

Comment (Aubron) [reply lib/index.js]: I like this approach, if there's a negative it's that someone adding additional functionality to the client wouldn't know to look for this abstraction and/or be aware that it's being applied, but I think it's straightforward enough to not be an issue.

Comment (dtolb) [reply lib/index.js]: A little readme explanation should cover it while we wait on typescript

**Pull request merged**